pages/models/generations/SM1/index.md

```diff
@@ -0,0 +1,170 @@
+
+
+<p float="center">
+<img src="{{'/pages/models/generations/SM1/SM_fixed_P.svg' | relative_url }}"
+     alt="Phasorial map of the operation with fixed active power"
+     style="width: 350px;" />
+
+<img src="{{'/pages/models/generations/SM1/SM_fixed_ef.svg' | relative_url }}"
+     alt="Phasorial map of the operation with fixed excitation"
+     style="width: 350px;" />
+
+</p>
+
+Left image shows different excitation levels for constant power operation. In normal excitation, $$\lvert e_f \rvert \cos(\delta) = \lvert v_t \rvert$$, and the current and voltage of the stator is in phase, meaning that the power factor $$\cos(\theta) = 1$$. In over or underexcitation, the current lags or leads the voltage respectively, providing or drawing reactive power from the grid respectively.
+
+Right image shows that for constant excitation, a greater torque angle $$\delta$$ improves the power factor. This torque angle is increased by increasing the load of the generator.
+
+### Operational limits
+
+Depending on the difference in phase between the rotor and grid rotation, the power transfered will be composed of active ($$P$$) and reactive ($$Q$$) power.
+
+These generated powers are limited by the heat limits of the components of the generator. The limits considered are the Armature and field current limit, due to Joule effect heating of both windings, and end region heating limit, which occurs due to currents in the structure of the stator when the field is underexcited.
+
+The following charts show the capability and compound curves for different power factors[[1]](#1). 
+
+<p float="center">
+<img src="{{'/pages/models/generations/SM1/SM_CapCurve.svg' | relative_url }}"
+     alt="Capability curve of a Synchronous Generator"
+     width="350px"/>
+<img src="{{'/pages/models/generations/SM1/SM_CompCurves.svg' | relative_url }}"
+     alt="Compound curves of a Synchronous Generator"
+     width="400px"/>
+</p>
+
+The left chart shows the $$P$$ and $$Q$$ maximum values for differnt limit curves, which depend on the refrigerator used, while the right chart shows the dependency on exciter current and apparent power for different values of the power factor.
+
+## Open source implementations
+
+This model has been successfully implemented in :
+
+
+| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
+| --------------| --- | --------- | ------------------- |------------------- | -------- |
+
+
+## Table of references
+
+<a id="1">[1]</a>  Kundur, Prabha. "Power System Stability and Control" New York, USA, 1994, McGraw-Hill
+
+<a id="2">[2]</a>  Kothari, D. P.; Nagrath, I. J. "Modern Power System Analysis", 4th ed., New Dehli, India, 2011, Tata McGraw-Hill
+
+<a id="3">[3]</a>  PowerWorld Corporation. "ECE 310 Syncrhonous Machine Modeling"
```

Comment:
you can add: Power 
System Dynamics and Stability by Peter Sauer and M.A. Pai from 1998

Reply:
added